tests/headers_test.cc

```diff
@@ -14,30 +14,30 @@ TEST(headers_test, accept)
 
     {
         const auto &media = a1.media();
-        ASSERT_EQ(media.size(), 1U);
```

Comment:
Why did you change the ASSERT_EQ() to ASSERT_TRUE().  The gtest suite prefers ASSERT_EQ().  Unit test failures have better error messages from that macro than from ASSERT_TRUE.  Here and in the rest of the unit tests.

Reply:
Hey @dennisjenkins75. See the commit log for an explanation. In short it's necessary for portability until gtest is fixed.

Reply:
The bug that you cited is from 2014, using googletest version 1.7.0.  It appears that pistache is also pulling in version 1.7.0.  However, the latest stable version is 1.8.1, released 2018-08-31 (per wikipedia).

Please revert the change to the unit tests, update googletest and try with 1.8.1  I am very hesitant to undo what is considered the proper usage of googletest.

Maybe update pistache to the latest stable googletest in a different PR.  Get that working cleanly, I'll merge it, then we can re-evaluate in this PR?

Reply:
I'm just reporting what I recall observing in the PPA sbuild environment. Even if we both have newer local distro packages, it won't change what the sbuild server had on it. I also confirmed the same problem on my local machine using 1.8.0.